rep-0152.rst

```diff
@@ -0,0 +1,144 @@
+REP: 152
+Title: Orphaned Pakcage Maintaining
+Author: Kei Okada
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 27-Dec-2017
+
+Outline
+=======
+
+#. Abstract_
+#. Motivation_
+#. Rationale_
+#. Operations_
+#. References_
+#. Copyright_
+
+Abstract
+========
+
+This REP specifies how we can organize and release packages without active maintainers.
+
+Motivation
+==========
+
+As the size of ROS ecosystem grows, there are several packages which loosing active maintainers.
+Main drawback of this situation is these packages do not get released into newer ROS distro coming every year, even if they are commonly used (For example, usb camera drivers are not released into Kinetic for more than half year [1]_ [2]_).
+Ideally everyone willing to use newer ROS distro almost equivalent to the set of packages in previous ROS distro.
+
+It is understandable that original maintainer could not continue serving as an active maintainer, because of starting new project, moving an institute/company, loosing interest and so on.
+And being a maintainer is not a easy job. It includes respond to incoming questions and issues, check for proposed changes and decide to include them to a main source tree and so on.
+
+The idea behind this document is to have a team of volunteers who can do ONLY the release job.
```

Comment:
I think it is crucial to mention that when doing a release of a package that person *must* also act on potential notification emails from the buildfarm. That implies to set a maintainer email which the person receives (as mentioned already). But I think the expectation to act on notification emails from the buildfarm should be explicitly stated - otherwise that burden falls onto the people operating the buildfarm which isn't acceptable imo.

Reply:
Sounds good to me.